server/src/routes/usersRouter.js

```diff
@@ -3,6 +3,10 @@ import { usersController } from "../controllers/usersController.js";
 
 const usersRouter = Router();
 
+usersRouter.get("/login/github", usersController.loginGithub);
+usersRouter.get("/login/github/callback", usersController.loginGithubCallback);
+usersRouter.delete("/email", usersController.deleteUserByEmail);
+usersRouter.delete("/github_id", usersController.deleteUserByGithubId);
```

Comment:
```suggestion
usersRouter.delete("/email/:email", usersController.deleteUserByEmail);
usersRouter.delete("/github_id/:githubId", usersController.deleteUserByGithubId);
```
보통 `get, delete` method는 body를 받기 보다는 앞에 params를 받도록 해서
```
req.params.email;
req.params.githubId;
```
이렇게 사용하는게 더 보편적입니다. (안되는 건 아니에요)

Reply:
한결 나아보입니다. 반영하겠습니다!